cloudferrylib/os/network/neutron.py

```diff
@@ -857,10 +916,23 @@ def upload_subnets(self, networks, subnets):
                 continue
             tenant_id = \
                 self.identity_client.get_tenant_id_by_name(snet['tenant_name'])
+            if not tenant_id:
```

Comment:
That's true. But from another side we can not check all such cases, cause our code will be grown up with huge amount of `if's`. Even in this module we have quite a few such situations, where we do not do checks...